### PR TITLE
Wow added!

### PR DESCRIPTION
**Pull request by foofx88**
To have more Wow when greeting!

### CODE REVIEW

File: intro.py [hunk 1]
```diff
@@ -9,6 +9,6 @@ def hello(first_name:str=None, last_name:str=None):
     elif not last_name:
         print(f'Hello {first_name}!')
     else:
-        print(f'Hello {first_name} {last_name}!')
+        print(f'Wow! {first_name} {last_name}!')
```
Comment (helenamin) [inline intro.py:12]: Previous code was better

**Review by Liutmantas — CHANGES_REQUESTED** [review]
lets replace wow with blah...

File: intro.py [hunk 2]
```diff
@@ -9,6 +9,6 @@ def hello(first_name:str=None, last_name:str=None):
     elif not last_name:
         print(f'Hello {first_name}!')
```
Comment (puneetverma) [inline intro.py:10]: good job!

**Review by puneetverma — APPROVED** [review]

**Comment by foofx88** [comment]
@Liutmantas, Please Specify Changes. If in regards of Wow, it will be disregarded

**Review by Liutmantas — APPROVED** [review]

**Pull request merged**